docs/Home.md

```diff
@@ -1,3 +1,50 @@
-Welcome to the android-templates wiki! 
+Welcome to the android-templates wiki!
 
-More coming soon...
\ No newline at end of file
+This repository generates a new project based on our preferences, by running a simple script. 
+
+Ex: `./newproject.sh -t crt -p co.myproject.example -n "MyProjectExampleCoroutine"`
+
+This script must include all essentials by default, while optional features can be appended with flags. To prevent this repository from becoming a dumping playground, features can be rejected too.
+
+Planning to contribute? Let's have a look at the criteria:
+
+How do we decide if a feature is **essential**? 👍
+
+- Our team requires it to maintain code quality.
+- Our team requires it to maintain a good developer experience.
+- Our team requires it to increase the productivity of the team.
+- It is always used or configured in the projects we've worked on.
+- It is recommended to use or configure based on Google's standards.
```

Comment:
I think we can reorder this a bit so it has a parallelism structure with the other option below. What I meant is that the first consideration must be about `how often it has been chosen or agreed to be picked` in a project.

The first 3 can be shortened to reduce the repetition in the writing, be precise.
>- Our team requires it to maintain code quality.
>- Our team requires it to maintain a good developer experience.
>- Our team requires it to increase the productivity of the team.

lastly, suggesting a more directive way of writing:
> It is a Google recommended component.

Reply:
@sleepylee Yes, there is some repetition: `Our team requires it to...` 

Do you suggest minimizing it in one sentence? I'm open to hearing your approach 🙏

Reply:
yes, I'd say we write it in a crystal-clear thought:
something like:


> - It is always implemented in the projects we have worked on or will work on.
> - It allows developers to increase the productivity from boilerplate setup, maintain the quality, and have a good experience for themselves.
>- It is a Google-recommended component.

what do you think?

Reply:
Thanks for the recommendations, I agree with the first and third bullet points.

For the second bullet point, we can't only refer to `developers`. 

Ex: `Chucker` or `Firebase App Distribution` also increases the productivity of our project managers.

Reply:
developers have the most benefit from it, in this codebase PM comes after so I would refrain to prioritize them in this topic, it will generate more conflict to resolve later when someone wants to bring something that has less value to dev but more value to the PM (btw, Product Manager, we don't have Project Manager role), how do you think?

Reply:
If I'm not mistaken, it was Julien that requested for Chucker.

As for Firebase App Distribution, is mostly beneficial for testers (our PM).

Please correct me if I got the wrong idea here 🙏

Reply:
nope, it was from our side to initialize Chucker after we learned from iOS. Eventually, we found it very useful for dev ourselves as well.

For Firebase App Distribution, it is to enhance our productivity because it's a requirement that often appears in our process, so we want to reduce the setup time. Again, the purpose of the feature is for everyone, the template is here to serve us: save our time on the setup.

Reply:
I agree, that it saves _our_ time on the setup. But both features were implemented for the team's purpose:

- We can check LogCat, so Chucker is not essential for us developers. It is essential for PMs however
- PMs require Firebase App Distribution for testing. So it's for their needs that we set it up 

🙏

Reply:
I still don't get your point from the latest argument. As been said, I don't deny that PM benefits from it, but I suggested to not involving them in this context because it can generate more debates in the future.

Now talking about Firebase or Chucker: 
- Do developers use them? Yes.
- Do we want to save time to set them up? Yes

So can we just talk about developers in here? I think yes too?

Reply:
> We can check LogCat, so Chucker is not essential for us developers. It is essential for PMs however.

LogCat is a console display, while Chucker allows you to debug while not connecting to AS. I guess you haven't had this experience before in AllOnline but we did have some good use cases with it in DM where we started using it more often. While it is not a blocker for our development, it has proved its usefulness to us, as well as the PM, so that's why it got promoted here to reduce the setup time.

Reply:
For firebase: to shape the understanding of "developer-oriented" way: it is a requirement that often come from the PM; while it benefits the PMs, we don't necessarily bring PM here as the reason. The real reason or the real benefit **to the developers** is that we don't have to re-setup it, again and again, all the time. Does this sound different from your original thought? 🤔